File: Modul 4/SOAL 5/PRAK305-2210817210002-GustiMuhammadNaufalRazin.py
from calendar import month_name


for i in range(5):
    s = int(input())
    dtk = s%60
    mnt = (s/60) % 60
    jam = (s/3600) %24
    h = s/86400

    if s<=3600 or s>3600 and s<=86400:
       print("%.2d:%.2d:%.2d\n" %(jam,mnt,dtk))
    elif s>86400:
        print("%d hari %.2d:%.2d:%.2d" %(h,jam,mnt,dtk))

File: Modul 4/SOAL 5/PRAK305-2210817210002-GustiMuhammadNaufalRazin.c
#include <stdio.h>
int main()
{
    for (int i=0; i<=5; i++){
        int s, dtk, mnt, jam, h;
        scanf("%d", &s);
        dtk = s%60;
        mnt = (s/60)%60;
        jam = (s/3600)%24;
        h  = s/84600;
        if(s<=3600 || s>3600 && s<=86400){
            printf("%.2d:%.2d:%.2d\n\n", jam, mnt, dtk);}
        else{
            printf("%d hari %.2d:%.2d:%.2d\n\n", h, jam, mnt, dtk);}
    }
}
